a10_octavia/controller/worker/tasks/decorators.py

```diff
@@ -14,14 +14,33 @@
 
 
 import acos_client
+from acos_client.errors import ACOSException
+from acos_client.errors import Exists
+
+try:
+    from http.client import BadStatusLine
+except ImportError:
+    from httplib import BadStatusLine
 from oslo_config import cfg
 from oslo_log import log as logging
 from requests.exceptions import ConnectionError
+from requests.exceptions import ReadTimeout
 
 CONF = cfg.CONF
 LOG = logging.getLogger(__name__)
 
 
+def configure_partition(vthunder_client, partition):
+    try:
+        try:
+            vthunder_client.system.partition.create(partition)
+        except Exists:
+            pass
+        vthunder_client.system.partition.active(partition)
+    except (ConnectionError, ACOSException, BadStatusLine, ReadTimeout):
```

Comment:
If we get an exception while trying to change the partition we shouldn't ignore it. Any calls made after this will lead to objects being created in the shared partition. We should 100% fail here.

Reply:
Addressed in https://github.com/a10networks/a10-octavia/pull/70